.github/workflows/merge.yml

```diff
@@ -130,11 +130,19 @@ jobs:
 
       - name: Run network tests (with encrypt-records)
         timeout-minutes: 25
-        run: cargo test --release --package sn_networking --features="open-metrics, encrypt-records"
+        run: cargo test --release --package sn_networking --features="open-metrics, encrypt-records" -- --skip can_store_after_restart
+
+      - name: Run network tests (with encrypt-records)
+        timeout-minutes: 5
+        run: cargo test --release --package sn_networking --features="open-metrics, encrypt-records" can_store_after_restart
```

Comment:
Why do we run this test separately?

Reply:
Strangely, I can run this `combined` with local machine.
However, the CI test always failed, which seems a parallel execution issue causing tests accessing same temp dir ?
Hence have to executed it separately.